pkg/controller/direct/alloydb/instance_fuzzer.go

```diff
@@ -50,14 +51,14 @@ func instanceFuzzer() fuzztesting.KRMFuzzer {
 	f.StatusField(".ip_address")
 	f.StatusField(".public_ip_address")
 	f.StatusField(".outbound_public_ip_addresses")
+	f.StatusField(".observability_config.track_wait_event_types")
 
 	f.Unimplemented_Internal(".satisfies_pzs")
 
 	f.Unimplemented_LabelsAnnotations(".labels")
 
+	f.Unimplemented_NotYetTriaged(".network_config")
```

Comment:
Network config is a supported field under spec: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/3e19bab7157c8e7074ee573dbbc7379b3e1f40be/apis/alloydb/v1beta1/instance_types.go#L92